build-tests/api-extractor-test-05/src/DocClass1.ts

```diff
@@ -22,9 +22,9 @@ export class DocClass1 {
 
   /**
    * This is an overloaded function.
-   * @param x - the number
-   * @param a - the first string
-   * @param b - the second string
+   * @param {x} - the number
```

Comment:
>{x}  [](start = 12, length = 4)

Don't use curly braces; in JSDoc they supply parameter type information, and in TSDoc they aren't used at alll